src/ambassadors/core.ts

```diff
@@ -45,7 +45,7 @@ const ambassadors = {
     scientific: "Dromaius novaehollandiae",
     sex: "Male",
     birth: "2021-02-14",
-    arrival: "2021-04",
+    arrival: "2021-06",
```

Comment:
```suggestion
    arrival: "2021-06-01",
```
Given we know it to be exactly June 1st, we can include that here